### PR TITLE
Fix/no header

### PR DESCRIPTION
**Pull request by lkappeler**

### CODE REVIEW

File: test/integration/TestDataProvider.js [hunk 1]
```diff
@@ -8,12 +8,17 @@ module.exports = class TestDataProvider {
                 {
                     name: 'testname Translation One',
                     description: 'Translation One testDescription',
-                    locale_id: 1,
+                    locale_id: 1, // de-ch from fixture
```
Comment (rcky) [inline test/integration/TestDataProvider.js]: Hmm, this is not necessarily id 1..... I think these tests are likely to break on other systems (e.g. if someone ran the migrations on the language-service multiple times and did not reset the index of the locale table).

File: test/integration/translationHandler.test.js [hunk 2]
```diff
@@ -142,6 +143,41 @@ describe('Translations Integration Test', () => {
             .get(getEndpoint)
             .set('Accept-Language', 'en-GB');
 
+        expect(getResponse.status).to.equals(200);
+        expect(getResponse.body).to.have.property('name', 'English Translation');
+        expect(getResponse.body).to.have.property('description', 'English testDescription');
+    });
+
+    it.only('returns empty translations for an non existing language aa-AA', async function() {
```
Comment (rcky) [inline test/integration/translationHandler.test.js]: Fixme: Remove only!

File: test/integration/translationHandler.test.js [hunk 3]
```diff
@@ -142,6 +143,41 @@ describe('Translations Integration Test', () => {
...
+        const getResponse = await this.service.api.request
+            .get(getEndpoint)
+            .set('Accept-Language', 'aa-AA');
+
+        expect(getResponse.status).to.equals(200);
+        expect(getResponse.body).to.have.property('name', '');
+        expect(getResponse.body).to.have.property('description', '');
+    });
+
+    it.only('returns empty translations if no language header is set', async function() {
```
Comment (rcky) [inline test/integration/translationHandler.test.js]: Fixme: Remove only!

File: test/integration/TestDataProvider.js (hunk 1 above)
Comment (lkappeler) [reply test/integration/TestDataProvider.js]: Actually I had the same in mind when I was looking at this but the locales are not loaded form the language service, they are provided as json fixture for the locale mode which uses the memory connector as a datasource.
https://github.com/joinbox/loopback-component-translations/blob/master/src/fixtures/locales.json
https://github.com/joinbox/loopback-component-translations/blob/master/test/support/server/datasources.json

**Pull request merged**